src/core/src/core_logic/Stopwatch.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+
+from core.src.bootstrap.Constants import Constants
+
+
+class Stopwatch(object):
+    """Implements the stopwatch logic"""
+
+    # Stopwatch exception strings
+    STARTED_ALREADY = "Stopwatch is already started"
+    NOT_STARTED = "Stopwatch is not started"
+    STOPPED_ALREADY = "Stopwatch is already stoppped"
```

Comment:
With the values growing, this could be an enum.

Reply:
Look for enumbackport in code to be python 2 compatible also